src/DataDefinitionsBundle/Importer/Importer.php

```diff
@@ -23,6 +23,10 @@
 use Pimcore\Model\Document;
```

Comment:
The importer should have no idea about the Process itself, it should be decoupled. Use events to communicate.

Reply:
I removed the code from Importer and Exporter.